tests/suites/ck/ck.sh

```diff
@@ -53,22 +48,23 @@ run_deploy_caas_workload() {
 
 	name="deploy-caas-workload"
 	k8s_cloud_name="k8s-cloud"
+	storage="csi-aws-ebs-default"
 	model_name="test-${name}"
 	file="${TEST_DIR}/${model_name}.log"
 
-	storage=$(kubectl get sc -o json | jq -r '.items[] | select(.metadata.annotations."storageclass.kubernetes.io/is-default-class"=="true") | .metadata.name')
-
 	controller_name=$(juju controllers --format json | jq -r '.controllers | keys[0]')
 	juju add-k8s "${k8s_cloud_name}" --storage "${storage}" --controller "${controller_name}" 2>&1 | OUTPUT "${file}"
 
 	add_model "${model_name}" "${k8s_cloud_name}" "${controller_name}" "${file}"
 
-	juju deploy cs:~juju/mariadb-k8s-3
-	juju deploy cs:~juju/mediawiki-k8s-4 --config kubernetes-service-type=loadbalancer
-	juju relate mediawiki-k8s:db mariadb-k8s:server
+	juju deploy postgresql-k8s
+	juju deploy mattermost-k8s
+	juju relate mattermost-k8s postgresql-k8s:db
```

Comment:
This is going to be really good for 3.1 as well :clap:

Reply:
Next step is to do a forward merge